schemas/input/csl-data.json

```
@@ -437,6 +437,21 @@
     "additionalProperties": false
   },
   "definitions": {
+    "rich-text-string": {
+      "title": "Rich Text String",
+      "description": "A string that may include sub-string formatting for bold, italic, subscript, superscript, smallcaps, and preserve case. In the JSON representation, these must be represented with well-formed HTML tags (b, i, sub, sup, and spans with 'small-caps' and 'preserve-case' class values, respectively). In the YAML representation, these must be represented by default with the equivalent markdown and pandoc extensions. For other representations, the 'markup' property should be used to indicate which format the processor should expect. TODO: math.",
```

```suggestion
      "description": "A string that may include sub-string formatting for bold, italic, subscript, superscript, smallcaps, and preserve case. In the JSON representation, these must be represented with well-formed HTML tags (b, i, sub, sup, a span with style='font-variant::small-caps', a span with class='nocase'); smallcaps and preserve case may also be represented with the non-standard HTML tags 'sc' and 'nc', respectively. In the YAML representation, these must be represented by default with the equivalent markdown (with pandoc extensions) syntax. For other representations, the 'markup' property should be used to indicate which format the processor should expect. TODO: math.",
```

`span class="nocase"` is what is currently used by [citeproc-js](https://www.zotero.org/support/kb/rich_text_bibliography) and [pandoc](https://pandoc.org/MANUAL.html#citations). `small-caps` isn't an HTML property, so well-formed HTML would need to be `span style="font-variant::small-caps"`. pandoc also supports `<sc> </sc>`. This shorthand is reasonable I think; similarly specifying `<nc> </sc>` as shorthand for nocase also seems reasonable

---

I _really_ dislike the idea of inventing our own syntax. 

The idea is (though I guess I forgot to include it) small caps would be represented using a `small-caps` span class:

``` html
<span class="small-caps">sc</span>
```

Regardless, others need to weigh in on this.